contracts/peg/PegStabilityModule.sol

```diff
@@ -322,11 +178,15 @@ contract PegStabilityModule is
         whileRedemptionsNotPaused
         returns (uint256 amountOut)
     {
-        amountOut = _redeem(to, amountFeiIn, minAmountOut);
+        amountOut = _redeem(to, amountVoltIn, minAmountOut);
     }
 
-    /// @notice function to buy FEI for an underlying asset
-    /// We first transfer any contract-owned fei, then mint the remaining if necessary
+    /// @notice function to buy VOLT for an underlying asset
+    /// This contract has no minting functionality, so the max
+    /// amount of Volt that can be purchased is the Volt balance in the contract
+    /// @param to recipient of the Volt
+    /// @param amountIn amount of underlying tokens used to purchase Volt
+    /// @param minAmountOut minimum amount of Volt recipient to receive
     function mint(
```

Comment:
TODO: Add test that combines a mint and redeem in the same transaction, and due to rounding profit is negative.